src/main/java/brooklyn/entity/proxy/aws/ElbControllerImpl.java

```diff
@@ -58,6 +58,9 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(ElbControllerImpl.class);
 
+    private ELBApi elbApi;
+    private LoadBalancerServiceContext loadBalancerServiceContext;
```

Comment:
What concerns me:
  * `loadBalancerServiceContext` is created
  * `elbApi` is created out of `loadBalancerServiceContext`
  * `loadBalancerServiceContext` is destroyed
  * what's the state of `elbApi` now? Don't think we should be using it at this point?

Moreover `loadBalancerServiceContext` is created and destroyed in each method - either create it once and re-use it together with `elbApi` or move them both as local variables.